web3rpc/rpc-specs/code-samples/curl/debug/inspection/gaslessInfo.sh

```diff
@@ -0,0 +1,9 @@
+curl -X 'POST' \
+  'https://public-en-kairos.node.kaia.io/kaia/sendRawTransaction' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "method": "debug_gaslessInfo",
+  "id": 1,
+  "jsonrpc": "2.0"
+}'
```

Comment:
1. RPC URLs are not supposed to have path `/kaia/sendRawTransaction`. See other curl examples such as https://github.com/kaiachain/kaia-sdk/blob/dev/web3rpc/rpc-specs/code-samples/curl/eth/block/blockNumber.sh
2. Missing `"params": []`